docs/release-notes/8.17.asciidoc

```diff
@@ -2,29 +2,44 @@
 == 8.17
 
 [discrete]
-[[release-notes-8.17.2]]
-=== 8.17.2
+[[release-notes-8.17.3]]
+=== 8.17.3
```

Comment:
Should we add a "Known issues" header under this version?

Reply:
Same comment as above.